### PR TITLE
I finally fixed all my CORS issues and my Proxy works.

### PR DESCRIPTION
**Pull request by hannahmanfredi**

### CODE REVIEW

File: proxy.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const myLimit = typeof(process.argv[2]) != 'undefined' ? process.argv[2] : '100kb';
+console.log('Using limit: ', myLimit);
+
+app.use(bodyParser.json({limit: myLimit}));
+
+app.use(express.static('public'));
+
+app.all('*', function (req, res, next) {
+
+    res.header("Access-Control-Allow-Origin", "*");
```
Comment (kimberlyjk) [inline proxy.js:17]: be sure to add this to your service as well.

File: proxy.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    res.header("Access-Control-Allow-Methods", "GET, PUT, PATCH, POST, DELETE");
+    res.header("Access-Control-Allow-Headers", req.header('access-control-request-headers'));
+
+    if (req.method === 'OPTIONS') {
```
Comment (kimberlyjk) [inline proxy.js:21]: I'm not 100% what this portion is checking..  Is it forwading the request or checking that there isn't an error?

**Review by kimberlyjk — APPROVED** [review]
this looks awesome! 

I'm not sure if you have added the CORS header allow access on your service, but I am getting an error that 
```
Access to script at 'http://localhost:3002/booking.js' from origin 'http://localhost:2000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```
I was able to get both overview and reviews to show up though! (I don't yet have the photogallery cloned onto my compuer. nice work!!

File: proxy.js (hunk 2 above)
Comment (hannahmanfredi) [reply proxy.js:21]: it's manually doing what the cors npm package does!

File: proxy.js (hunk 1 above)
Comment (hannahmanfredi) [reply proxy.js:17]: yup it's there using default app.use(cors())

**Pull request merged**